docs/modules/ROOT/pages/tutorials/getting-started.adoc

```diff
@@ -0,0 +1,34 @@
+= Getting Started with APPUiO Cloud
+
+This tutorial describes your first steps with APPUiO Cloud, from signup to your first deployment.
+
+== Step 1: Create an Account
+
+To create your APPUiO Cloud account, fill the form https://www.appuio.ch/en/offering/cloud/[in this page]. You will receive an e-mail inviting you to create a password for your newly created account.
+
+== Step 2: Login
```

Comment:
Why is this step needed? What does it do?

Reply:
Because where else could the user see the list of (and quickly access to) available zones?

Reply:
Ok, let's leave it like this